docs/src/main/asciidoc/native-and-ssl-guide.adoc

```diff
@@ -65,9 +65,12 @@ They enable the native SSL support for your native executable.
 
 As SSL is ipso facto the standard nowadays, we decided to enable its support automatically for some of our extensions:
 
- * the SmallRye REST client extension,
- * the Agroal connection pooling extension,
- * the Jaeger extension.
+ * the REST client extension (`quarkus-rest-client`),
+ * the Agroal connection pooling extension (`quarkus-agroal`),
+ * the Jaeger extension (`quarkus-jaeger`),
+ * the mailer extension (`quarkus-mailer`)
+ * the Neo4j extension (`quarkus-neo4j`).
```

Comment:
Neo4j was an example, there are a lot more missing. Just look for references to `ExtensionSslNativeSupportBuildItem`.

As for the Elasticsearch client extension, better name the Hibernate Search + Elasticsearch one because that's the one we want to advertise.